test/client_server_SUITE.erl

```diff
@@ -83,3 +99,24 @@ basic_push(_Config) ->
     ct:pal("Response Body: ~p", [ResponseBody]),
     13 = length(Streams),
     ok.
+
+send_body_opts(_Config) ->
+    {ok, Client} = http2_client:start_link(),
+    RequestHeaders =
+        [
+         {<<":method">>, <<"GET">>},
+         {<<":path">>, <<"/index.html">>},
+         {<<":scheme">>, <<"https">>},
+         {<<":authority">>, <<"localhost:8080">>},
+         {<<"accept">>, <<"*/*">>},
+         {<<"accept-encoding">>, <<"gzip, deflate">>},
+         {<<"user-agent">>, <<"chattercli/0.0.1 :D">>}
+        ],
+
+    ExpectedResponseBody = <<"BodyPart1\nBodyPart2">>,
+
+    {ok, {ResponseHeaders, ResponseBody}} = http2_client:sync_request(Client, RequestHeaders, <<>>),
```

Comment:
There's a better way to test this using `http2c`. It will allow you to inspect the response as a series of frames. You can then assert that the response is made of two data frames, and that they're broken up correctly.